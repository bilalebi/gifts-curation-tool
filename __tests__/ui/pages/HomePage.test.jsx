import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import HomePage from '../../../src/ui/Home';

describe('HomePage component', () => {
  test('should render', () => {
    const component = renderer
      .create(<MemoryRouter><HomePage /></MemoryRouter>)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
