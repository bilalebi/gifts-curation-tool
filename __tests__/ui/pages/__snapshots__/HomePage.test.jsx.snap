// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render 1`] = `
<main>
  <div
    className="home-banner"
  >
    <div
      className="column medium-offset-3 medium-6 text-center"
    >
      <h5>
        Search for a mapping:
      </h5>
      <form
        className="input-group"
        onSubmit={[Function]}
      >
        <input
          className="input-group-field"
          placeholder="ENST00000620613, A7E2Y1, ..."
          type="text"
        />
        <div
          className="input-group-button"
        >
          <button
            className="button"
            type="submit"
          >
            Submit
          </button>
        </div>
      </form>
      <div
        className="home-banner__actions"
      >
        <button
          className="button"
          onClick={undefined}
        >
          Explore Mappings
        </button>
      </div>
    </div>
  </div>
</main>
`;
